### PR TITLE
docs: add user guide for the re-labeling feature

### PR DESCRIPTION
**Pull request by eloy-encord**

### CODE REVIEW

File: docs/docs/user-guide/relabeling.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 6
+---
+
+# Re-labeling
+
+**Learn how to send your data to labeling in Encord Annotate**
```
Comment (frederik-encord) [inline docs/docs/user-guide/relabeling.md:7]:
```suggestion
**Learn how to send your data for labeling in Encord Annotate**
```

File: docs/docs/user-guide/relabeling.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Steps to send data to the labeling stage
+
+0. **[Optional]** Use the [filter feature](../user-guide/filtering) from the toolbox in the _Data Quality_ or _Label Quality_ _Explorer_ section to choose the desired data. 
+1. Access the Action tab from the toolbox in the _Data Quality_ or _Label Quality_ _Explorer_ section.
+2. Click the _🖋 Re-label_ button.
+3. Verify the number of tasks ready for submission to the labeling stage is correct and press the _Confirm_ button.  
+4. **[Optional]** Navigate to the project in Encord Annotate and explore the task modifications in the _Summary_ tab.
+
+![action-tab-in-toolbox](../images/user-guide/action-tab-in-toolbox.png)
+
```
Comment (frederik-encord) [inline docs/docs/user-guide/relabeling.md:22]: Would be nice with a short text and a screendump of what to expect in the Annotate platform after clicking the button.

File: docs/docs/user-guide/relabeling.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+When examining your project data, you might come across labels that appear incorrect or are missing altogether.
+With Encord Active, you have the ability to highlight such data and seamlessly transfer it to Encord Annotate, at the dedicated labeling stage for your project.
+This empowers annotators to address any missing elements and enhance the overall quality of your labels, ensuring a more accurate and comprehensive data annotation process.
+
+## Steps to send data to the labeling stage
+
+0. **[Optional]** Use the [filter feature](../user-guide/filtering) from the toolbox in the _Data Quality_ or _Label Quality_ _Explorer_ section to choose the desired data. 
+1. Access the Action tab from the toolbox in the _Data Quality_ or _Label Quality_ _Explorer_ section.
+2. Click the _🖋 Re-label_ button.
+3. Verify the number of tasks ready for submission to the labeling stage is correct and press the _Confirm_ button.
```
Comment (frederik-encord) [inline docs/docs/user-guide/relabeling.md:18]:
```suggestion
3. Verify the number of frames for submission to the labeling stage is correct and press the _Confirm_ button.  
```

**Review by frederik-encord — CHANGES_REQUESTED** [review]

File: docs/docs/user-guide/relabeling.md (hunk 2 above)
Comment (eloy-encord) [reply docs/docs/user-guide/relabeling.md:22]: haha, yes, just waiting to build the confirmation button first 🦺 and add the image later

**Review by frederik-encord — APPROVED** [review]

**Pull request merged**